src/InventoryManager.java

```diff
@@ -0,0 +1,263 @@
+                    if((parts == null) || (parts.length != 4)) {
+                        System.out.println("Invalid inventory");
+                        continue;
+                    }
+                    String name = parts[0];
+                    String price = parts[1];
+                    String quantity = parts[2];
+                    String category = parts[3];
+
+                    if(!isInteger(quantity)) {
+                        System.out.println("Invalid quantity");
+                        continue;
+                    }
+                    if(!isDouble(price)) {
+                        System.out.println("Invalid price");
+                        continue;
+                    }
+
+                    Inventory inventory = createItem(name, Double.parseDouble(price), Integer.parseInt(quantity), category);
+                    if(itemMap.containsKey(name)) {
+                        System.out.println(name + " Already created.");
+                        continue;
+
+                    }
+                    else {
+                        itemMap.put(name, inventory);
+
+                    }
+
+                }
+
+                // sell an item thus reducing the value of quatity
+                else if(option.equals("2")) {
+                    System.out.println("Enter comma delimiited name,quantity");
+                    String line = br.readLine();
+                    String[] parts = line.split(",");
+                    if((parts == null) || (parts.length != 2)) {
+                        System.out.println("Invalid inventory");
+                        continue;
+                    }
+                    String name = parts[0];
+                    String quantity = parts[1];
+
+                    if(!isInteger(quantity)) {
+                        System.out.println("Invalid quantity");
+                        continue;
+                    }
+                    if(!itemMap.containsKey(name)) {
+                        System.out.println(name + " Does not exist.");
+                        continue;
+                    }
+                    Inventory inventory = itemMap.get(name);
+                    int i = inventory.getQuantity();
+                    if(Integer.parseInt(quantity) > i) {
+                        throw new Exception("Inventory Quantity less than " + quantity);
+                    }
+                    else {
+                        inventory.setQuantity(inventory.getQuantity() - Integer.parseInt(quantity));
+                    }
+                }
+
+                // remove item from map
+                else if(option.equals("3")) {
+                    System.out.println("Enter name");
+                    String line = br.readLine();
+                    String[] parts = line.split(",");
+                    if((parts == null) || (parts.length != 1)) {
+                        System.out.println("Invalid name");
+                        continue;
```

Comment:
There is a large number of `continue` statements.  Moving forward this should serve as a red flag that the flow of the program is a bit rough and might be able to be refactored.